tests/test_xtks_calendar.py

```diff
@@ -160,6 +160,26 @@ def regular_holidays_sample(self, silver_week_holidays, golden_week_holidays):
             "2021-11-23",  # Labor Thanksgiving Day
             "2021-12-31",  # Market Holiday
             #
+            # 2022
+            "2022-01-01",  # New Year's Day
```

Comment:
@beppe712 can you please check the https://github.com/AlpacaDB/trading_calendars/pull/22#issuecomment-982181293 and add the csv file ("tests/resources/xtks.csv") as it is in the #22.

Reply:
Done 😊 54a8688c3722ff0ae168a498ad8deaba6e0c7551